apps/openmw/mwclass/container.cpp

```diff
@@ -282,9 +282,11 @@ namespace MWClass
         if (ptr.getCellRef().getTrap() != "")
             text += "\n#{sTrapped}";
 
-        if (MWBase::Environment::get().getWindowManager()->getFullHelp()) {
-            text += MWGui::ToolTips::getCellRefString(ptr.getCellRef());
+        if (MWBase::Environment::get().getWindowManager()->getFullHelp())
+        {   text += MWGui::ToolTips::getCellRefString(ptr.getCellRef());
             text += MWGui::ToolTips::getMiscString(ref->mBase->mScript, "Script");
+            if (Misc::StringUtils::ciEqual(ptr.getCellRef().getRefId(), "stolen_goods"))
+                text += "\nYou can not use evidence chests";
```

Comment:
What's the point of this warning?

Reply:
When user uses TFH + owned crosshair, tooltip shows ranks requirements, which player may pass, while crosshair always shows that using evidence chest is not permitted. It may confuse user and considered to be a bug.